library/core/src/cell/once.rs

```diff
@@ -6,7 +6,7 @@ use crate::{fmt, mem};
 /// This allows obtaining a shared `&T` reference to its inner value without copying or replacing
 /// it (unlike [`Cell`]), and without runtime borrow checks (unlike [`RefCell`]). However,
 /// only immutable references can be obtained unless one has a mutable reference to the cell
-/// itself. In the same vein, the cell can only be re-initialized with such a mutable reference.
+/// itself. In the same vein, the cell can only be re-set with such a mutable reference.
```

Comment:
Instead of ".. re-set .." I'd like to see something like this here: can only be set and unset (with `take`).